test/models/customer_test.rb

```diff
@@ -0,0 +1,51 @@
+require "test_helper"
+
+describe Customer do
+  describe "relations" do
+    let(:current_customer) { customers(:customer1)}
+
+    it "can get the list of rentals from 'rentals'" do
```

Comment:
You should have negative and positive cases of each relationship (with rentals and without).